codemod_yaml/box/yaml/sequence.py

```diff
@@ -51,3 +77,8 @@ def set_value(self, other: BoxedPy) -> None:
         self._value = other
 
     value = property(get_value, set_value)
+
+    @property
+    def end_byte(self) -> int:
+        # TODO conditional
+        return self.node.end_byte + 1
```

Comment:
This is to include the trailing `\n` which isn't owned by the BlockSequenceItem (it is an unowned byte within the enclosing BlockSequence)